man/import.q.feedback.Rd

```diff
@@ -30,20 +35,22 @@
 		A matrix with short handles as rownames, languages in columns, as produced by \code{\link{build.q.set}}.
 	}
 	\item{manual.lookup}{
-		A matrix with meaningful short handles (same as in \code{\link{build.q.set}}, \code{\link{import.q.concourse}}) as rownames, and arbitary, unique identifying strings in any of the columns as printed on the q-cards by \code{\link{make.cards}}.
-		Defaults to NULL in which case items are automatically identified by full item hashes, as also used by \code{\link{make.cards}}.
+		A matrix with meaningful short handles (same as in \code{\link{build.q.set}}, \code{\link{import.q.concourse}}) as rownames, and arbitary, unique identifying strings (IDs) in any of the columns as printed on the q-cards by \code{\link{make.cards}}.
+		Defaults to \code{NULL} in which case items are automatically identified by full item hashes, as also used by \code{\link{make.cards}}.
 	}
 }
 
 \details{
-	Participants in q studies are often invited to provide open-ended feedback on items, given q-researchers a additional information on participants' viewpoints.
-	This feedback is conveniently entered in a spreadsheet editor (2nd column), where each line of feedback corresponds to an item id (1st column), either a \code{manual.lookup}, or an automatic hash from \code{\link{make.cards}}.
-	An additional (3rd) column indicates whether the current line should be ignored (TRUE), as may be the case for privacy reasons or when the feedback is merely a correction.
+	Participants in Q studies are often invited to provide open-ended feedback on items, given Q researchers a additional information on participants' viewpoints.
+	This feedback is conveniently entered in a spreadsheet editor (2nd column), where each line of feedback corresponds to an item ID (1st column), either a \code{manual.lookup}, or an automatic hash from \code{\link{make.cards}}.
+	An additional (3rd) column indicates whether the current line should be ignored (TRUE), as may be the case for privacy reasons or when the feedback is merely a correction of a typographic error.
 
-	After an (arbitrary) header line, a csv may look like this:
-	\code{c12,"This q-item sounds like r-research to me!",FALSE}, indicating that it should \emph{not} be ignored (FALSE).
+	After an (arbitrary) header line, a CSV may look like this:
+	\code{c12,"This q-item sounds like r-research to me!",FALSE}, indicating that it should \emph{not} be ignored (\code{FALSE}).
 
 	Item feedback is best enclosed in quotes, allowing comas inside cells.
+
+	More details on the files and directory structure expected by this function can be found in the \link{qmethod-package} documentation, and on the package \href{https://github.com/aiorazabala/qmethod/wiki/Data-management}{wiki}.
 }
 
 \value{
```

Comment:
In the latest version, I can see some more text:

`Returns a matrix (with a single condition) or array (when more than one condition) with handles as row names, people as column names, conditions (if more than one) as 3rd dimname and ranks in cells [what does 'ranks in cells' mean?]. The return parallels the output from import.q.sorts, but with feedback as array cells, rather than ranks [Q sort scores?].`